nixos/modules/services/printing/cupsd.nix

```diff
@@ -185,6 +190,17 @@ in
         '';
       };
 
+      printersConf = mkOption {
+        type = types.nullOr types.lines;
+        default = null;
+        description = ''
+          Contents of the printers configuration file of the CUPS daemon
+          (<filename>printers.conf</filename>). When left null, cups will use
+          the (<filename>/etc/cupsd/printers.conf</filename>) as the configuration
```

Comment:
ah yes, `cups` was meant. Will fix the doc. As to the behavior, as with other configuration files, as soon as the configuration is modified through the web interface, cups will move `printers.oonf` aside to `printers.conf.0`. Otherwise, cups will use the declarative file. This seems to be consistent with
the behavior of other configurations files.

It is indeed desirable to me, not sure about others. I can live with rebasing the changeset over 17.03 though as long as it is available in the next release (17.09?).